docs/sources/next/results-output/web-dashboard/_index.md

```diff
@@ -65,6 +65,12 @@ To automatically generate a report from the command line once the test finishes
 K6_WEB_DASHBOARD=true K6_WEB_DASHBOARD_EXPORT=html-report.html k6 run script.js
 ```
 
+{{< admonition type="note" >}}
+
+The report only includes graphs if the test duration is greater than three times the aggregation period value, set by the `K6_WEB_DASHBOARD_PERIOD` variable.
```

Comment:
@szkiba I updated the explanation based on your comment, and also switched it to an admonition so it's easier for readers to see this.

What do you think? 🤓

Reply:
Perfect :) 
maybe I would mention 30sec as the default value in parentheses, just so the user doesn't have to search)